doc/source/configuration/config_api.rst

```diff
@@ -255,7 +255,8 @@ Runs the service using the local service account.
 Custom Install Actions
 ======================
 
-These settings allow user-specified code to be executed during the service install/uninstall process.
+These settings allow user-specified code to be executed during the service install/uninstall process. Each action can take an instance of `Topshelf.Runtime.InstallHostSettings` in the parameter, allowing the user to leverage the settings defined for the service, such as the service name, description, etc.
```

Comment:
The argument type is actually just `Topshelf.HostSettings`.

Reply:
Also, if you could rebase this into a single commit (git rebase -i HEAD~3, I think) - however many commits there are to squash into a single commit. That would be great. Thank you!